GVFS/GVFS.Virtualization/Projection/GitIndexProjection.cs

```diff
@@ -691,17 +675,18 @@ private static List<ProjectedFileInfo> ConvertToProjectedFileInfos(SortedFolderE
             {
                 childEntry = sortedFolderEntries[i];
 
-                if (childEntry.IsIncluded)
+                if (childEntry.IsFolder)

```

Comment:
Perhaps `if ((childEntry as FolderData)?.IsIncluded == true)`? Maybe that's too terse and the nested `if` is better.

Reply:
Out of curiosity, how would you expect the performance of `as` to compare to the virtual `IsFolder` call?

Reply:
Running a simple test and `as` was in most cases about 2x slower but I had to up the iterations in the test to at least a million before it was even taking 1 ms so unless we are using it in a tight loop I'm sure other code will be a bottleneck before this.

Reply:
Well, thanks for checking the perf! That would have been a mistake to listen to me.